pennylane_cirq/cirq_device.py

```diff
@@ -155,14 +158,29 @@ def __init__(self, wires, shots, analytic, qubits=None):
     }
 
     _observable_map = {
-        "PauliX": None,
-        "PauliY": None,
-        "PauliZ": None,
-        "Hadamard": None,
+        "PauliX": CirqOperation(lambda: cirq.X),
+        "PauliY": CirqOperation(lambda: cirq.Y),
+        "PauliZ": CirqOperation(lambda: cirq.Z),
+        "Hadamard": CirqOperation(lambda: cirq.H),
         "Hermitian": None,
-        "Identity": None,
+        "Identity": CirqOperation(lambda: cirq.I),
     }
 
+    def to_paulistring(self, observable):
+        """Convert an observable to a cirq.PauliString"""
+        if isinstance(observable, Tensor):
+            obs = [self.to_paulistring(o) for o in observable.obs]
+            return functools.reduce(operator.mul, obs)
+
+        cirq_op = self._observable_map[observable.name]
+        if cirq_op is None:
+            raise NotImplementedError(f"{observable.name} is not currently supported.")
```

Comment:
I assume this catches `qml.Hermitian`?

Out of curiosity, are there any observables cirq supports that PL doesn't?

Reply:
We _could_ use `qml.utils.decompose_hamiltonian()` to convert the Hermitian matrix to a Pauli string, but not sure if it's worth it

Reply:
I can add a TODO